### DISCUSSION TITLE
linkToDataTable высота окна

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#495] [ОШБК!] [answered]
В коде действия открываю временную таблицу:
```
= : linkToDataTable(table: 'nps_answer'; title: str`'Результаты опроса по группе: ' + #title`; width: "70vw"; height: "60vh"; params: $par; topbuttons: false; bottombuttons: false; target: "iframe"; refresh: false)
	par: rowCreate(field: "h_group" = #id)
```

Но параметр height: "60vh" не отрабатывает. Окно всегда открывается с одной и той же высотой, пробовал и в пикселях и в процентах. В чем может быть проблема?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Вроде работает:

```
= : linkToDataTable(table: 'raschet_dostavki'; title: "*"; width: 800; height: "30vh"; refresh: false)
```

* px не указываются если в пикселях то `height: 300;`

<img width="1230" alt="Снимок экрана 2023-12-19 в 14 44 07" src="https://github.com/totumonline/totum-ru-issues-and-discussions/assets/55755565/dcf19708-35b6-4f9a-aceb-b01125959959">

**Comment by Spyker086** [comment]
Но у нас как будто вообще не реагирует на параметр height. И в пикселях и в % по разному пробовал
